app/models/student.rb

```diff
@@ -0,0 +1,20 @@
+class Student < ActiveRecord::Base
+	belongs_to:standard
+	#validates:enrollment_no,presence:true,uniqueness:true
+	validates:first_name,presence:true
+	validates:middle_name,presence:true
```

Comment:
No need to specify each parameter in different line for same type of validation.

Ex. `validates :first_name, :middle_name, :last_name, ...  , presence: true`